docs/190-paas-operations/004-architecture.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: cluedin
+nav_order: 4
+parent: PaaS operations
+permalink: /deployment/architecture
+title: Architecture overview
+tags: ["deployment", "ama", "marketplace", "azure", "architecture"]
+last_modified: 2023-11-08
+headerIcon: "paas"
+---
+## On this page
+{: .no_toc .text-delta }
+1. TOC
+{:toc}
+
+# High Level Design
```

Comment:
Please remove this section heading. There is no need for it when there is just one section.